graphqlapi/src/main/resources/schemafirst/cql_directives.graphql

```diff
@@ -235,6 +247,23 @@ directive @cql_where (
     predicate: Predicate = EQ
 ) on ARGUMENT_DEFINITION
 
+"""
+Annotates a parameter to customize the DELETE condition that is generated from it.
```

Comment:
I think we need to be more detailed here. As much as "WHERE condition" should be clear to everyone because of the similarity with SQL, conditional updates are more specific to CQL.

Proposal:
```suggestion
Annotates a parameter to indicate that it will be used as a condition that must test true on the selected entity in order for the mutation to be applied.

This is only allowed for delete and update mutations.
```